### PR TITLE
Bug 1358631 - Take info bar heights into account when determining if a job is in view

### PR DESCRIPTION
**Pull request by KWierso**

### CODE REVIEW

File: ui/js/directives/treeherder/clonejobs.js [hunk 1]
```diff
@@ -818,8 +818,16 @@ treeherder.directive('thCloneJobs', [
 
         var isOnScreen = function(el){
             var viewport = {};
-            viewport.top = $(window).scrollTop() + $("#global-navbar-container").height();
-            viewport.bottom = viewport.top + $(window).height() - $("#info-panel").height();
+            viewport.top = $(window).scrollTop() + $("#global-navbar-container").height() + 20;
```
Comment (KWierso) [inline ui/js/directives/treeherder/clonejobs.js]: This and the "- 20" down below are to make sure if a job partially overlaps the header or info-panel elements, the job fully scrolls into view.

File: ui/js/directives/treeherder/clonejobs.js [hunk 2]
```diff
@@ -818,8 +818,16 @@ treeherder.directive('thCloneJobs', [
...
+            var filterbarheight = $(".active-filters-bar").height();
+            if (filterbarheight > 0) {
```
Comment (camd) [inline ui/js/directives/treeherder/clonejobs.js]: could be:

    viewport.top = filterbarheight ? viewport.top + filterbarheight : viewport.top;

File: ui/js/directives/treeherder/clonejobs.js [hunk 3]
```diff
@@ -818,8 +818,16 @@ treeherder.directive('thCloneJobs', [
...
+                viewport.top = viewport.top + filterbarheight;
+            }
+            var updatebarheight = $(".update-alert-panel").height();
+            if (updatebarheight > 0) {
```
Comment (camd) [inline ui/js/directives/treeherder/clonejobs.js]: same as above

File: ui/js/directives/treeherder/clonejobs.js (hunk 2 above)
Comment (KWierso) [inline ui/js/directives/treeherder/clonejobs.js]: These actually need to check for being greater than zero. When I calculated the height while the bars weren't shown, they were coming out as something like -3.something.

**Pull request merged**